src/directives/avatar.ts

```diff
@@ -183,6 +182,7 @@ export default [
                         // Otherwise, if we requested a high res avatar but
                         // there is only a low-res version, show that.
                         if (this.highResolution
+                            && this.isLoading
```

Comment:
This is the reason why the low-res version was displayed instead of the fallback.